### PR TITLE
social links fixed text

### LINKED ISSUE #4609: Forms: Social Media - Only request to fill in short form, e.g. username for link specification
Make social media links on all forms in a way that users only need to add their username or other short forms.

Relevant Forms:
* Wizard step two social links
* Attendee Forms
* Speaker submission forms
* Speaker profile form

E.g. show as follows
Twitter https://twitter.com/___________
Github https://github.com/___________
Linkedin https://www.linkedin.com/____________
Instagram https://instagram.com/______________
Facebook https://facebook.com/_______________
YouTube https://youtube.com/________________

Screenshot from 2020-07-22 17-20-27

### PR DESCRIPTION
**Pull request by ruhci28**
Fixes #4609

#### Short description of what this resolves:
make social media inputs so that user needs to fill their username . 

#### Changes proposed in this pull request:

-add if cases for each social link 
-
-

#### Checklist

- [x ] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [ x] My branch is up-to-date with the Upstream `development` branch.
- [ x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ x
Screenshot 2020-07-27 at 7 03 10 PM
Screenshot 2020-07-25 at 5 59 43 PM

] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by ruhci28** [comment]
Hii @snitin315  can you give suggestion now .?

File: app/templates/components/widgets/forms/link-input.hbs [hunk 1]
```diff
@@ -25,6 +31,60 @@
       {{/if}}
     </div>
   </div>
+{{else if (eq this.inputId 'speaker_twitter')}}
```
Comment (snitin315) [inline app/templates/components/widgets/forms/link-input.hbs:34]: `speaker_twitter` will be specefic to speaker form only, I think we can normazlize it like below. Can you try if something like below is working?

```suggestion
{{else if this.inputId.includes('twitter')}}
```

File: app/templates/components/widgets/forms/link-input.hbs [hunk 2]
```diff
@@ -25,6 +31,60 @@
...
+    <div class="menu">
+      <div class="item" data-value="https://facebook.com/">https://facebook.com/</div>
+    </div>
+  </UiDropdown>
+  <Input @type="text" @value={{this.address}} @name={{name}} @id={{this.inputId}} />
+{{else if (eq this.inputId 'speaker_instagram')}}
+  <UiDropdown @class="label" @selected={{'https://instagram.com/'}} @onChange={{action (mut this.protocol)}}>
+    <div class="text">https://instagram.com/</div>
+    <i class="dropdown icon"></i>
+    <div class="menu">
```
Comment (snitin315) [inline app/templates/components/widgets/forms/link-input.hbs:65]: I beleive removing this `menu` div will get the desired result.

File: app/templates/components/widgets/forms/link-input.hbs (hunk 1 above)
Comment (ruhci28) [reply app/templates/components/widgets/forms/link-input.hbs:34]: Ok

File: app/templates/components/widgets/forms/link-input.hbs (hunk 2 above)
Comment (ruhci28) [reply app/templates/components/widgets/forms/link-input.hbs:65]: Ok i will try it .

**Comment by ruhci28** [comment]
Screenshot 2020-07-28 at 9 10 39 AM
Hii @snitin315 removing the div giving the desired result but includes function is giving this error.
Screenshot 2020-07-28 at 9 11 46 AM

**Comment by snitin315** [comment]
Then we need to find a better way , to apply this widget to all github,twitter etc fields

**Comment by ruhci28** [comment]
Does we have to made a  helper to check substring is included in a string

**Comment by snitin315** [comment]
Not yet, need to create one.

**Comment by ruhci28** [comment]
Hii @snitin315 i created a helper to detect a substring is present in a string or not . 
Screenshot 2020-07-28 at 12 03 39 PM
and use that helper like this in else if 
Screenshot 2020-07-28 at 12 04 00 PM
but it is giving an error 
Screenshot 2020-07-28 at 12 04 11 PM
can you suggest any change or how should i use the helper .

**Comment by snitin315** [comment]
See how parameters are passed in other helpers.

**Comment by snitin315** [comment]
`identifySocialLinks([mainString, subString])` For additional info refer https://guides.emberjs.com/release/components/helper-functions/

**Comment by ruhci28** [comment]
Hii @snitin315 I was calling the function incorrectly now the function is working fine and giving expecting result
Screenshot 2020-07-28 at 9 17 52 PM
 .

**Comment by snitin315** [comment]
Please push your changes.

**Comment by ruhci28** [comment]
yeah sure

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- app/helpers/identify-social-link.js  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5916026&bid=19383059)

**Comment by ruhci28** [comment]
Hii @snitin315  can review it

**Comment by snitin315** [comment]
Now you need to check  if it's storing correct value in db. If not we may need to modify `computed-helpers.js`

**Comment by ruhci28** [comment]
Hii @snitin315 how will i check the storing value in db . did you give any suggestion on how should i proceed
.

**Comment by snitin315** [comment]
> Hii @snitin315 how will i check the storing value in db . did you give any suggestion on how should i proceed
> .

Just test your feature that it is working properly as expected.

**Pull request closed without merging**